### ISSUE TITLE
Debian-based Linux distributions adopt PEP 668

### ISSUE THREAD

**Issue by alembcke** [ManimCommunity/manim#3260]
## Description of proposed feature

Now that Debian-based Linux distributions have adopted [PEP 668](https://peps.python.org/pep-0668/), a .deb package should probably be created for Manim and maintained with Debian maintainers.

## How can the new feature be used?

Having a Debian package installable via `apt` would make installing Manim on Debian much easier.

## Additional comments

I can give this one a go, but I have never contributed to a Linux distribution, so it might take me a while.  Also, I want to check that others agree that this is a good idea?

As long as I don't get any objections, I will update this issue with my progress.

**Comment by alembcke** [comment]
It looks like @roehling has already created an ITP (Intent To Package):

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1012608

**Comment by roehling** [comment]
That is correct. I have packaged most of the dependencies already. Still missing is skia-pathops, which depends on Google Skia, which is a bit more involved to package.

**Comment by alembcke** [comment]
> That is correct. I have packaged most of the dependencies already. Still missing is skia-pathops, which depends on Google Skia, which is a bit more involved to package.

Great!  Let me know if there is anything I can do to help.  I am new to contributing packages to Debian (or any Linux distro), but am willing to learn.

**Comment by roehling** [comment]
I'm happy to help if you'd like to get your feet wet with Debian packaging! Feel free to drop me an email and I'll point you to some helpful tutorials and sponsor your uploads when the time comes.
